_includes/configuration/advtable_value_series.md

```diff
@@ -0,0 +1,91 @@
+## `advtable_value_series`
+{{ site.requires_5_9v }}
```

Comment:
```suggestion
## `advtable_value_series`

{{ site.requires_5_9v }}
```
I'm not too sure if this would cause issues, but elsewhere we keep a blank line between these to make sure they don't "merge"